kedro-datasets/kedro_datasets/api/api_dataset.py

```diff
@@ -34,88 +39,89 @@ class APIDataSet(AbstractDataSet[None, requests.Response]):
     data_catalog.html#use-the-data-catalog-with-the-code-api>`_:
     ::
 
-        >>> from kedro_datasets.api import APIDataSet
+        >>> from kedro.extras.datasets.api import APIDataSet
```

Comment:
Saw this has already been merged but I think this got copied over by mistake, I'll change it back in #189